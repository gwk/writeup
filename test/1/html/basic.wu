writeup v0

# Title

# Section 1
Paragraph 1 line 1.
Paragraph 1 line 2.
`code span`. <b: bold text>. <span: style=color:red; span style red>.

Paragraph 2.

List:
* A
* B
  * B1
  * B2
* C
Post-list.

## Section 1.1
Line 2.
Line 3.


# Section 2: Quote
> # Header
> Text.
> List:
> * A
>   * B
>
>> Double quote.
>| Quoted code.

# Section 3: Code
| # Header 1
| Line 1.
| Line 2. `code span`.
