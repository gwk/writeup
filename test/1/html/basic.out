<section class="S1" id="s0">
  <h1 id="h0">Title</h1>
</section>
<section class="S1" id="s1">
  <h1 id="h1">Section 1</h1>
  <p>
    Paragraph 1 line 1.
  <br />
    Paragraph 1 line 2.
  <br />
    <code class="inline">code&nbsp;span</code>. <b>bold text</b>. <span style="color:red">span style red</span>.
  </p>
  <p>
    Paragraph 2.
  </p>
  <p>
    List:
  </p>
  <ul class="L1">
    <li>A</li>
    <li>
      <p>
        B
      </p>
      <ul class="L2">
        <li>B1</li>
        <li>B2</li>
      </ul>
    </li>
    <li>C</li>
  </ul>
  <p>
    Post-list.
  </p>
  <section class="S2" id="s1.1">
    <h2 id="h1.1">Section 1.1</h2>
    <p>
      Line 2.
    <br />
      Line 3.
    </p>
  </section>
</section>
<section class="S1" id="s2">
  <h1 id="h2">Section 2: Quote</h1>
  <blockquote>
    <section class="S1" id="q1s0">
      <h1 id="h0">Header</h1>
      <p>
        Text.
      <br />
        List:
      </p>
      <ul class="L1">
        <li>
          <p>
            A
          </p>
          <ul class="L2">
            <li>B</li>
          </ul>
        </li>
      </ul>
      <blockquote>
        <p>
          Double quote.
        </p>
      </blockquote>
<div class="code-block">
<code class="line">Quoted code.</code>
</div>
    </section>
  </blockquote>
</section>
<section class="S1" id="s3">
  <h1 id="h3">Section 3: Code</h1>
<div class="code-block">
<code class="line"># Header 1</code>
<code class="line">Line 1.</code>
<code class="line">Line 2. `code span`.</code>
</div>
</section>
